auth-web/src/composables/affiliations-factory.ts

```diff
@@ -157,10 +157,12 @@ export const useAffiliations = () => {
     )
   }
 
-  /** Returns the Name Request type using the NR type code */
+  /** Returns the Name Request type using the NR action code or the NR type code */
   const nameRequestType = (business: Business): string => {
     if (isNameRequest(business)) {
-      const nrType = CommonUtils.mapRequestTypeCdToNrType(business.nameRequest?.requestTypeCd)
+      // Try action code first, and if not found in the enum then use type code
+      let nrType = CommonUtils.mapRequestActionCdToNrType(business.nameRequest?.requestActionCd)
+      nrType = nrType || CommonUtils.mapRequestTypeCdToNrType(business.nameRequest?.requestTypeCd)
```

Comment:
The types of requestActionCd and requestTypeCd aren't the same but since they're used for a display string, this is not a problem, though you might want to type `nrType` as string just to emphasize this.